_health-care/_js/utils/validations.js

```diff
@@ -12,7 +12,12 @@ function isValidDate(day, month, year) {
     date.getFullYear() === Number(year);
 }
 
+function isValidPhone(value) {
+  return /^\d{3}-\d{3}-\d{4}$/.test(value);
```

Comment:
TODO this for now, but I wonder if we should just use a library like
  https://www.npmjs.com/package/validator
or
  https://www.npmjs.com/package/iz

both seem to be relatively well used. Especially the first...(36,287 downloads in the last day!)